src/shared/views/BuildingFootprint/Validator.js

```diff
@@ -2,145 +2,300 @@
 import * as React from 'react';
 import { compose } from 'redux';
 import { connect } from 'react-redux';
+import get from 'lodash.get';
+import pako from 'pako';
+import base64 from 'base-64';
 import { firebaseConnect, isEmpty, isLoaded } from 'react-redux-firebase';
-import { StyleSheet, View } from 'react-native';
+import { Image, StyleSheet, Text, View } from 'react-native';
 import Button from 'apsl-react-native-button';
 import FootprintDisplay from './FootprintDisplay';
 import LoadingIcon from '../LoadingIcon';
-import { COLOR_GREEN, COLOR_RED, COLOR_YELLOW } from '../../constants';
+import { COLOR_WHITE } from '../../constants';
+import GLOBAL from '../../Globals';
 
 import type {
     BuildingFootprintGroupType,
-    ProjectType,
     BuildingFootprintTaskType,
+    ResultMapType,
+    SingleImageryProjectType,
 } from '../../flow-types';
 
+// in order to allow enough screen height for satellite imagery on small
+// screens (less than 550px high) we make buttons smaller on those screens
+const buttonHeight = GLOBAL.SCREEN_HEIGHT >= 550 ? 50 : 40;
+const buttonMargin = GLOBAL.SCREEN_HEIGHT >= 550 ? 5 : 3;
+
+const buttonBGColor = 'rgba(255, 255, 255, 0.2)';
+const buttonBGColorSelected = 'rgba(255, 255, 255, 0.8)';
+
 const styles = StyleSheet.create({
-    button: {
-        borderRadius: 0,
-        height: 55,
-        marginBottom: 0,
-        marginTop: 5,
+    container: {
+        alignItems: 'center',
+        flex: 1,
+        flexDirection: 'column',
+        justifyContent: 'flex-start',
+    },
+    checkmark: {
+        alignSelf: 'center',
+        marginBottom: -3,
+        height: 25,
+        width: 25,
+    },
+    sideBySideButtons: {
+        flexDirection: 'row',
+        justifyContent: 'space-between',
+    },
+    bigSquareButton: {
+        borderColor: COLOR_WHITE,
+        borderRadius: 20,
+        borderWidth: 2,
+        height: buttonHeight * 2,
+        flex: 1,
+        marginBottom: buttonMargin,
+        marginLeft: buttonMargin,
+        marginRight: buttonMargin,
+        marginTop: buttonMargin,
+    },
+    bigSquareButtonText: {
+        alignSelf: 'center',
+        color: COLOR_WHITE,
+        fontSize: 18,
+        fontWeight: 'bold',
+    },
+    longNarrowButton: {
+        borderColor: COLOR_WHITE,
+        borderRadius: 20,
+        borderWidth: 2,
+        color: COLOR_WHITE,
+        height: buttonHeight,
+        marginBottom: buttonMargin,
+        marginLeft: buttonMargin,
+        marginRight: buttonMargin,
+        marginTop: buttonMargin,
+    },
+    longNarrowButtonText: {
+        color: COLOR_WHITE,
+        fontWeight: 'bold',
     },
 });
 
-const FOOTPRINT_CORRECT = 1;
-const FOOTPRINT_NEEDS_ADJUSTMENT = 2;
-const FOOTPRINT_NO_BUILDING = 3;
+const FOOTPRINT_NO_BUILDING = 0;
```

Comment:
I would rename this to FOOTPRINT_YES, FOOTPRINT_NO

Reply:
makes sense. fixed.